Source/Tools/FEXInterpreter/CMakeLists.txt

```diff
@@ -5,50 +5,56 @@ if (ENABLE_VIXL_SIMULATOR)
   list(APPEND DEFINES -DVIXL_SIMULATOR=1)
 endif()
 
-add_executable(FEXInterpreter
+add_executable(FEX
   FEXInterpreter.cpp
   AOT/AOTGenerator.cpp)
 
-target_compile_definitions(FEXInterpreter PRIVATE ${DEFINES})
+target_compile_definitions(FEX PRIVATE ${DEFINES})
 
-# Enable FEX APIs to be used by targets that use target_link_libraries on FEXInterpreter
-set_target_properties(FEXInterpreter PROPERTIES
+# Enable FEX APIs to be used by targets that use target_link_libraries on FEX
+set_target_properties(FEX PROPERTIES
   ENABLE_EXPORTS 1
   C_VISIBILITY_PRESET hidden
   CXX_VISIBILITY_PRESET hidden
   VISIBILITY_INLINES_HIDDEN TRUE
 )
 
-target_include_directories(FEXInterpreter
+target_include_directories(FEX
   PRIVATE
     ${CMAKE_CURRENT_SOURCE_DIR}/
     ${CMAKE_BINARY_DIR}/generated
 )
-target_link_libraries(FEXInterpreter
+target_link_libraries(FEX
   PRIVATE
     ${LIBS}
     LinuxEmulation
     CommonTools
     ${PTHREAD_LIB}
     fmt::fmt
 )
-target_compile_options(FEXInterpreter PRIVATE ${FEX_TUNE_COMPILE_FLAGS})
+target_compile_options(FEX PRIVATE ${FEX_TUNE_COMPILE_FLAGS})
 
 if (CMAKE_BUILD_TYPE MATCHES "RELEASE")
-  target_link_options(FEXInterpreter
+  target_link_options(FEX
     PRIVATE
       "LINKER:--gc-sections"
       "LINKER:--strip-all"
       "LINKER:--as-needed"
   )
 endif()
 
-install(TARGETS FEXInterpreter
+install(TARGETS FEX
   RUNTIME
     DESTINATION bin
     COMPONENT Runtime
 )
 
+# Create a copy of FEX with legacy names until phased out.
+install(PROGRAMS ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/FEX
+  RENAME FEXInterpreter
+  DESTINATION bin
+  COMPONENT Runtime)
```

Comment:
Perhaps use a `LegacyRuntime` component for this? It'll still be installed with `ninja install` then, but everyone who already migrated can easily opt out of the redundant binary.